examples/servers_detail_export_xlsx.py

```diff
@@ -2,167 +2,181 @@
+                vulnerabilities_tab.write(row + 1, col + 7, cve.cvss.authentication)
+                vulnerabilities_tab.write(row + 1, col + 8, cve.cvss.availability_impact)
+                vulnerabilities_tab.write(row + 1, col + 9, cve.cvss.confidentiality_impact)
+                vulnerabilities_tab.write(row + 1, col + 10, cve.cvss.integrity_impact)
+
+            if cve.cvss_v3 is not None:
+                vulnerabilities_tab.write(row + 1, col + 11, cve.cvss_v3.access_vector)
+                vulnerabilities_tab.write(row + 1, col + 12, cve.cvss_v3.access_complexity)
+                vulnerabilities_tab.write(row + 1, col + 13, cve.cvss_v3.privileges_required)
+                vulnerabilities_tab.write(row + 1, col + 14, cve.cvss_v3.user_interaction)
+                vulnerabilities_tab.write(row + 1, col + 15, cve.cvss_v3.integrity_impact)
+                vulnerabilities_tab.write(row + 1, col + 16, cve.cvss_v3.availability_impact)
+                vulnerabilities_tab.write(row + 1, col + 17, cve.cvss_v3.confidentiality_impact)
+                vulnerabilities_tab.write(row + 1, col + 18, cve.cvss_v3.scope)
+
+            row += 1
+        row += 2
+
+def recommended_actions_details(recommended_tab, servers):
+    """Write each recommended actions/affected technologies of each Host in `Recommended Actions` tab"""
+    row = 0
+    col = 0
+    for server in servers:
+        recommended_tab.write(row, col, server.hostname)
+        row += 1
+
+        recommended_tab.write(row, col, "Affected technology")
+        recommended_tab.write(row, col + 1, "CVE announcement")
+        recommended_tab.write(row, col + 2, "Patchable")
+        recommended_tab.write(row, col + 3, "Current version")
+        recommended_tab.write(row, col + 4, "Target version")
+
+        for update in server.updates:
+            if update.current is not None:
+                recommended_tab.write(row + 1, col, update.current.product)
+                recommended_tab.write(row + 1, col + 3, update.current.version)
+                recommended_tab.write(row + 1, col + 1, ', '.join(update.cve_announcements))
+                recommended_tab.write(row + 1, col + 2, update.patchable)
+                recommended_tab.write(row + 1, col + 4, update.target.version)
+            else:
+                recommended_tab.write(row + 1, col, update.target.product)
+                recommended_tab.write(row + 1, col + 1, ', '.join(update.cve_announcements))
+                recommended_tab.write(row + 1, col + 2, update.patchable)
+
+            row += 1
+        row += 2
+
+
+def launch_script():
+    '''Launch script'''
+    client = connect_api()
+    file = instantiate_export()
+    export_xls(file, client)
+    print("INFO: Done.")
+
+def main(args=None):
+    '''Main function'''
+
+    parser = argparse.ArgumentParser(
+        description="Script using Cyberwatch API to export vulnerabilities of servers with details to XLSX")
```

Comment:
add this comment at the top of the script (all scripts should have a comment defining what they do). Remove argparse usage if we do not offer any arguement.